### PR TITLE
#11017 Log In & Password protected videos in iframe

### PR DESCRIPTION
**Pull request by mreyk**

### CODE REVIEW

File: src/Pumukit/JWPlayerBundle/Controller/BasePlayerController.php [hunk 1]
```diff
@@ -24,7 +24,8 @@ class BasePlayerController extends BasePlayerControllero implements WebTVControl
     public function indexAction(MultimediaObject $multimediaObject, Request $request)
     {
         $embeddedBroadcastService = $this->get('pumukitschema.embeddedbroadcast');
-        $response = $embeddedBroadcastService->canUserPlayMultimediaObject($multimediaObject, $this->getUser(), $request->headers->get('PHP_AUTH_PW'), $request->query->get('force-auth'));
+        $password = $request->get('broadcast_password');
```
Comment (rubenrua) [inline src/Pumukit/JWPlayerBundle/Controller/BasePlayerController.php]: Why not only password? KISS

File: src/Pumukit/WebTVBundle/Resources/views/Index/401unauthorized.html.twig [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+    <form name="passwdForm" method="post" action="{{ path('pumukit_videoplayer_index', {'id': mm.id})|raw}}">
+      <div class="col-md-12">
+        <input id="broadcast_password" name="broadcast_password" type="password" placeholder="{% trans %}Introduce a password{% endtrans %}..." value="" size="17" maxlength="256" style="margin-top: 5px;margin-left:5px; margin-right: 5px; margin-bottom: 10px; width: 150px;"/>
+      </div>
+      <div class="col-md-12 text-center">
+        <span id="player_password_check_fail" style="color: red; margin: 0px 10px;{{ invalid_password ? '':'display: none;' }}">{% trans %}Password is not valid{% endtrans %}.</span>
+      </div>
+      {% set diff = (app.request.getQueryString) ? '&' : '?' %}
+      <div class="col-md-12">
+        <button type="submit" class="btn btn-primary btn-flat" style="margin-top:10px;" name="submit" title="Log In" onclick="location.href='{{ app.request.requestUri }}{{ diff }}broadcast_password=test'">enter pass</button>
```
Comment (rubenrua) [inline src/Pumukit/WebTVBundle/Resources/views/Index/401unauthorized.html.twig]: Why `broadcast_password=test `?

File: src/Pumukit/WebTVBundle/Resources/views/Index/401unauthorized.html.twig [hunk 3]
```diff
@@ -1,12 +1,28 @@
...
+</h1>
+<div class="grid_10 mm_player">
+  <div class="date"></div>
+  <h2 style="text-align: center;">{% trans %}This video is private with restricted access{% endtrans %}.</h2>
+</div>
+<div class="grid_5">
+  <h3 class="text-center">{% trans %}Enter a valid password to watch it{% endtrans %}.</h3>
+  <br/>
+  <div class="text-center">
+    <form name="passwdForm" method="post" action="{{ path('pumukit_videoplayer_index', {'id': mm.id})|raw}}">
```
Comment (rubenrua) [inline src/Pumukit/WebTVBundle/Resources/views/Index/401unauthorized.html.twig]: Note: try using `action="#"`.

**Comment by rubenrua** [comment]
@mreyk code review my updates.

**Comment by mreyk** [comment]
@rubenrua all check, it seems perfect.

**Pull request merged**